test/rake_ext_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class RakeExtTest < ActiveSupport::TestCase
+  test "can exclude rake tasks" do

```

Comment:
Can't get this test to work, RakeExt is causing an infinite loop.
```ruby
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
```

Reply:
@carl-printreleaf can you try this approach https://gist.github.com/igorkasyanchuk/b828f483d0d132c1ca43733b101ce5e0

it works locally, but maybe you can check it too and prepare a PR

Reply:
@igorkasyanchuk thanks for the reply, I'll try it out and update my pr if it works

Reply:
@igorkasyanchuk your suggestion worked. I updated the pr. Let me know if you have any other comments.